src/components/Shared/MinimalSpacingTooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactNode } from 'react';
+import Tooltip from '@mui/material/Tooltip';
+
+interface MinimalSpacingTooltipProps {
```

Comment:
Sorry to end up rewriting this whole thing. But you can just do this to pass through all props to the tooltip and also add on `PopperProps`.

```tsx
import React from 'react';
import { Tooltip, TooltipProps } from '@mui/material';

const MinimalSpacingTooltip: React.FC<TooltipProps> = (props) => (
  <Tooltip
    PopperProps={{
      modifiers: [
        {
          name: 'offset',
          options: {
            offset: [0, -5],
          },
        },
      ],
    }}
    {...props}
  />
);

export default MinimalSpacingTooltip;
```